cmd/keytransparency-client/cmd/root.go

```diff
@@ -44,6 +43,7 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/credentials/oauth"
+	"github.com/google/trillian/crypto/keys/pem"
```

Comment:
Looks like this file needs a `gofmt` to keep the imports alphabetized 
I've just configured `gometalinter` for this project.  You might enjoy linking it up to your IDE.